.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-**Prep Release: (assigned to:)**
-_Due: <3-business-days-before-release>_
-- [ ] Merge all features and fixes into the `dev` branch
-- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
-- [ ] Update dependencies and increment extension version in the `dev` branch.
-- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
-- [ ] Close out or punt remaining GitHub issues for the current milestone
+**Prep DTFx Release: (assigned to:)**
+_Due: <2-3-business-days-before-release>_
+- [ ] Increment updated DTFx packages (either DT-AzureStorage only or DT-AzureStorage, DT-Core, and DT-Redis)
+- [ ] Delete DTFx test packages from myget.
```

Comment:
Sometimes a DTFx package might be on myget for a customer, shouldn't we have an exception for that case? Maybe we can delete the myget packages post-nuget release

Reply:
We should be using staging for release prep, and the main appservice feed for test packages for customers.

Reply:
Let's be explicit about which myget feed we delete/upload to.

Reply:
I'm not sure if this has changed, but the MyGet *staging* feed had some automatic cleanup processes before based on Functions release schedules. Given we don't release at the same cadence as the Functions team, we may want to be careful about taking a dependency on their staging feed.

Reply:
@cgillum Good point. Can we create our own myget feed? Or should we just talk to the functions team about leaving our packages out of their automatic cleanup (if possible)?

Reply:
I'll talk with Fabio's team regarding the deletion. I believe it is manual cleanup, but I can verify.